_includes/nav.html

```diff
@@ -1,55 +1,100 @@
 <ul class="nav-list">
-  {%- assign ordered_pages_list = site.html_pages | where_exp:"item", "item.nav_order != nil" -%}
-  {%- assign unordered_pages_list = site.html_pages | where_exp:"item", "item.nav_order == nil" -%}
+  {%- assign included_pages = site.html_pages
```

Comment:
```suggestion
  {%- assign included_pages = include.pages
```

Reply:
@kevinlin1 thanks for this correction - I'd completely forgotten about the support for using collections!

@pmarsceill I'd better add navigation tests for the collections feature, using a separate config file for activating the tests.

Reply:
@pmarsceill I've tested locally that collections navigation works after making the (already committed) correction suggested by @kevinlin1. 

However, I'm **not** committing the tests, because they involve renaming the `docs` folder to `_docs`, and uses of `link` tags then need adjusting... 

For the record, I tested locally by moving `_docs/tests` to `_tests`, and I used the following in `_config.yml`:
```yaml
collections:
  docs:
    output: true
  tests:
    output: true

just_the_docs:
  collections:
    docs:
      name: Documentation
    tests:
      name: Tests
```

BTW, I tried making `_docs` a symlink to `docs`, to avoid the need for renaming `docs`, but Jekyll complained about watching the same directory more than once. There's an explanation of the causes of such error reports at https://github.com/guard/listen/wiki/Duplicate-directory-errors. Unfortunately I couldn't find a Jekyll config option to avoid the error report. And in any case, I'm not a big fan of symlinks.